yarn.lock

```diff
@@ -2,22 +2,17 @@
 # yarn lockfile v1
```

Comment:
I updated a lot of versions.

Reply:
There are 3,738 changes for the lock file which sounds like a lot for installing one package and updating another one. Are you sure it is accurate?

Reply:
It seems like a lot was just out of date. delete yarn lock and then did `yarn install`